lst_scripts/batch_dl1_utils-merge_and_copy.py

```diff
@@ -0,0 +1,44 @@
+#!/usr//bin/env python3
+
+# E. Garcia, 12/02/2020
+
+import argparse
+from .data_management import move_dir_content, check_and_make_dir_without_verification
```

Comment:
`from .data_management` will not work.
Plus, with installed packaged, you might prefer `from lst_scripts.data_management import ... `

Reply:
Right for the `.data_management import ...`. It was set like this because pycharm (my local program) doesn't like find the path otherwise. So they must be changed in every file.